src/ripe_rainbow/interactive/drivers.py

```diff
@@ -227,16 +227,9 @@ def click(self, element, ensure = True):
         return element
 
     def ensure_visible(self, element, timeout = None):
-        # waits until the element is visible from a document point of view
-        # no scroll ensuring a z index collision is enforced
-        self._wait(timeout = timeout).until(
-            lambda d: element.is_displayed(),

```

Comment:
On Firefox `is_displayed` does not seem obey the same definition as Chrome. In this case, for a visible (display != none && opacity > 0) element that is outside the viewport, it returns false.

Because of this, I think we can let go of this safeguard, as there isn't a very big reason to fork the behavior depending on the browser.